centos/8/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM centos:8
+MAINTAINER Alexey Kopytov <akopytov@gmail.com>
+
+# Fix missing locales
+ENV LANGUAGE="en_US.UTF-8"
+
+# Enable extra repositories
+RUN yum -y install \
+    wget \
+    yum-utils \
+    epel-release
```

Comment:
Should not we also enable PowerTools?

Reply:
I truly do not know. Just scratching my own itch here, and the current image works for me.

Reply:
It is recommended to enable PowerTools in the EPEL repository instructions: https://github.com/packpack/packpack-docker-images/pull/39#discussion_r338324713

It also seems logical to have built tools enabled in packpack images (because they are for building something).